.github/workflows/genbuilds.xsh

```diff
@@ -23,5 +23,8 @@ for os_name, python_version in product(OS_NAMES, PYTHON_VERSIONS):
     s = template.replace("OS_NAME", os_name)
     s = s.replace("OS_IMAGE", OS_IMAGES[os_name])
     s = s.replace("PYTHON_VERSION", python_version)
+    # Allow Python 3.9 to fail.
+    if python_version in ["3.9"]:
+        s = "\n".join((s, "        continue-on-error: true\n"))
```

Comment:
There should probably be a global set of allowed failures and then this would check if it is in that set, for maintainability. Thanks!

Reply:
Added :+1: